pkgs/os-specific/linux/nvidia-x11/settings.nix

```diff
@@ -43,6 +43,14 @@ let
 
     makeFlags = [
       "OUTPUTDIR=." # src/libXNVCtrl
+      "libXNVCtrl.a"
+      "libXNVCtrl.so"
```

Comment:
Since there's a merge conflict anyway, another nit: `makeTargets` is more explicit

Reply:
I tried `makeTargets` first, and it did not work.  I don't see any mention of this attribute in the [nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-makeFlags).

Reply:
Hm, I was sure I saw the attribute being used somewhere, but I don't see any mentions in `pkgs/stdenv/generic/setup.sh` so I must be wrong